src/OrchardCore/Orchard.DisplayManagement/Fluid/FluidViewTemplate.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Fluid;
+using Fluid.Values;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.FileProviders;
+using Newtonsoft.Json.Linq;
+using Orchard.DisplayManagement.Fluid.Filters;
+using Orchard.DisplayManagement.Fluid.Internal;
+using Orchard.DisplayManagement.Fluid.ModelBinding;
+using Orchard.DisplayManagement.Shapes;
+using Orchard.Settings;
+
+namespace Orchard.DisplayManagement.Fluid
+{
+    public class FluidViewTemplate : FluidTemplate<ActivatorFluidParserFactory<FluidViewParser>>
+    {
+        public static readonly string ViewsFolder = "Views";
+        public static readonly string ViewExtension = ".liquid";
+        private static readonly MemoryCache Cache = new MemoryCache(new MemoryCacheOptions());
+
+        static FluidViewTemplate()
+        {
+            TemplateContext.GlobalMemberAccessStrategy.Register<JObject>((obj, name) => obj[name]);
+
+            if (!FluidValue.TypeMappings.TryGetValue(typeof(JObject), out var value))
+            {
+                FluidValue.TypeMappings.Add(typeof(JObject), o => new ObjectValue(o));
+                FluidValue.TypeMappings.Add(typeof(JValue), o => FluidValue.Create(((JValue)o).Value));
+            }
+
+            if (!FluidValue.TypeMappings.TryGetValue(typeof(DateTime?), out value))
```

Comment:
Yes, because a nullable so not a basic type, but i don't need it anymore.

First i was using something like this `ContentItem.CreatedUtc.Value`. But now i use a filter `ContentItem.CreatedUtc | date_time ...`. See the `date_time` filter where our custom `DateTime Shape` is used, as done in the original `TheBlogTheme`.

Just removed and it works.